### PR TITLE
fix some comment

### PR DESCRIPTION
**Pull request by zhiqiangxu**

### CODE REVIEW

File: mempool/mempool.go [hunk 1]
```diff
@@ -927,8 +927,7 @@ func (mp *TxPool) validateReplacement(tx *btcutil.Tx,
 func (mp *TxPool) maybeAcceptTransaction(tx *btcutil.Tx, isNew, rateLimit, rejectDupOrphans bool) ([]*chainhash.Hash, *TxDesc, error) {
 	txHash := tx.Hash()
 
-	// If a transaction has witness data, and segwit isn't active yet, If
-	// segwit isn't active yet, then we won't accept it into the mempool as
+	// If a transaction has witness data, and segwit isn't active yet, then we won't accept it into the mempool as
```
Comment (guggero) [inline mempool/mempool.go:930]: Please break after 80 characters (while using 8 spaces for any tab character)

File: txscript/standard.go [hunk 2]
```diff
@@ -153,7 +153,7 @@ func isPubKeyScript(script []byte) bool {
 // is a standard pay-to-pubkey-hash script.  It will return nil otherwise.
 func extractPubKeyHash(script []byte) []byte {
 	// A pay-to-pubkey-hash script is of the form:
-	//  OP_DUP OP_HASH160 <20-byte hash> OP_EQUALVERIFY OP_CHECKSIG
+	//  OP_DUP OP_HASH160 OP_DATA_20 <20-byte hash> OP_EQUALVERIFY OP_CHECKSIG
```
Comment (guggero) [inline txscript/standard.go:156]: The `OP_DATA_20` is kind of implied by the `<20-byte hash>`...

**Review by guggero — COMMENTED** [review]
Thanks. Can you please squash your commits into a single one and use a proper commit message?

File: txscript/standard.go (hunk 2 above)
Comment (zhiqiangxu) [reply txscript/standard.go:156]: It's not consistent, [here](https://github.com/btcsuite/btcd/blob/b1b94202082b8e22160df31349992f37fb360f75/txscript/standard.go#L99) is an example.

File: mempool/mempool.go (hunk 1 above)
Comment (zhiqiangxu) [reply mempool/mempool.go]: Done.

**Comment by zhiqiangxu** [comment]
> Thanks. Can you please squash your commits into a single one and use a proper commit message?

Done.

**Comment by guggero** [comment]
Still two commits and the message should say what the commit changes, not what the developer did when creating the commit.

**Comment by zhiqiangxu** [comment]
> Still two commits and the message should say what the commit changes, not what the developer did when creating the commit.

Another squash is done:)

**Review by guggero — APPROVED** [review]
Thanks, LGTM :tada:

**Review by jcvernaleo — APPROVED** [review]
OK

**Pull request merged**